extensions/wikia/MyHome/ActivityFeedTag.php

```diff
@@ -53,6 +51,13 @@ function ActivityFeedTag_render($content, $attributes, $parser, $frame) {
 		)
 	);
 
+	$style = getStyle($parameters);
+
 	wfProfileOut(__METHOD__);
 	return "<div$style>$feedHTML</div>$snippets";
 }
+
+function getStyle( $parameters ) {
+	$style = $parameters['style'];
+	return empty($style) ? '' : ' style="' . $style . '"';
```

Comment:
You have changed `empty($parameters['style'])` to less safe `$style = $parameters['style'];` and `empty($style)`. The key `style` is not something mandatory. The `$parameters` array might simply not have that key in which case the assignment will result with a PHP Warning. And in this case, the warning is not something that needs consideration.